File: Proyecto/server/EntityLayer/EstadoProcesoEntity.py
from datetime import datetime
from pydantic import BaseModel
from Utilidades.Enumerado.ProcessActionEnum import ProcessActionEnum

class EstadoProcesoSaveModel(BaseModel):
    EstadoProcesoId: int 
    Valor: int 
    Nombre: str 
    Descripcion: str 
    FechaRegistro: datetime 
    CodUsuario: str 
    EstadoRegistro: bool 
    Action: ProcessActionEnum

class EstadoProcesoItemModel:
    EstadoProcesoId: int 
    Valor: int 
    Nombre: str 
    Descripcion: str 
    FechaRegistro: datetime 
    CodUsuario: str 
    EstadoRegistro: bool 

    def Cargar(_DB):
        c =  EstadoProcesoItemModel()
        c.EstadoProcesoId = _DB["EstadoProcesoId"] 
        c.Valor = _DB["Valor"] 
        c.Nombre = _DB["Nombre"] 
        c.Descripcion = _DB["Descripcion"] 
        c.FechaRegistro = _DB["FechaRegistro"] 
        c.CodUsuario = _DB["CodUsuario"] 
        c.EstadoRegistro = bool(ord(_DB["EstadoRegistro"])) 
        return c

File: ProyectoMysql/Api/LogisticStorage/LogisticStorage.EntityLayer/EstadoProcesoEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LogisticStorage.EntityLayer
{
    public class EstadoProcesoEntity : BaseEntityObject
    {

        public EstadoProcesoEntity()
        {
            this.EstadoProcesoId=0;
            this.Valor = 0;
            this.Nombre = String.Empty;
            this.Descripcion = String.Empty;
            this.FechaRegistro = DateTime.Now;
            this.CodUsuario = String.Empty;
            this.EstadoRegistro = false;
        }

        public Int32 EstadoProcesoId { get; set; }
        public Int16 Valor { get; set; }
        public String Nombre { get; set; }
        public String Descripcion { get; set; }
        public DateTime FechaRegistro { get; set; }
        public String CodUsuario { get; set; }
        public Boolean EstadoRegistro { get; set; }

    }
}
